### PR TITLE
Ddt 1342 get party

### PR DESCRIPTION
**Pull request by FinnNielsen**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ services:
     ports:
       - 8443:8443
     environment:
-      - "db_hostname=dcsa-ec-registry-test-db"
```
Comment (pimg) [inline docker-compose.yml:11]: does the [docker-compose-two-platforms.yml](https://github.com/dcsaorg/DCSA-EBL-Envelope/blob/master/docker-compose-two-platforms.yml) also need to change?

**Review by pimg — CHANGES_REQUESTED** [review]
does not need to go into master as discussed during refinement, need to go in a separate branch for the next phase. branch could be named "next-phase".

File: docker-compose.yml (hunk 1 above)
Comment (FinnNielsen) [reply docker-compose.yml:11]: I don't think so, it has it's own db

**Comment by FinnNielsen** [comment]
Changed to merge into "next-phase"

File: docker-compose.yml (hunk 1 above)
Comment (FinnNielsen) [reply docker-compose.yml:11]: docker-compose-two-platforms.yml is now aligned with docker-compose.yml

**Review by pimg — APPROVED** [review]

**Pull request merged**